datetimerange/__init__.py

```diff
@@ -702,6 +702,28 @@ def truncate(self, percentage):
         self.__start_datetime += discard_time
         self.__end_datetime -= discard_time
 
+    def union(self, x):
+        """
+        Returns the timedelta of the union of the input and current 
+        DateTimeRange.
+        
+        :param DateTimeRange x: List of DateTimeRanges.
```

Comment:
`x` is not a List of `DateTimeRange` instances judging from what the method does.

Reply:
@atoaster
Thank you for your PR.

The method looks like useful for some use cases.
What concerns me is that the method name deviates from what the method does.
The method return value is not returning a union of `DateTimeRange` but `timedelta`.
I think that `union_timedelta` or something would be more suitable name for the method.